default.vcl

```diff
@@ -202,4 +202,16 @@ sub vcl_deliver {
     } else {
         set resp.http.X-Cache = "MISS";
     }
+
+    # CORS response for smartlogic widget
+    if (req.http.Origin == "https://cloud.smartlogic.com" && req.url ~ "(\/content|\/concept\/search|\/concordances)") {
```

Comment:
Will this actually match those endpoints? Or do you need `.*` at the end of each regex? Furthermore, the `/concept/search` endpoint is the `POST` one and not the `GET` based one (that's `/concepts`); is that the intention?

Reply:
Well I think this matches endpoints. I've checked this regex on [here](https://fiddle.fastlydemo.net/fiddle/aa4eeb66), but adding `.*` would be safer.

Yes, `POST` endpoint for `/concept/search` is intentional. I'd like to use this endpoint with keyword based search, so `POST` looks fine https://github.com/Financial-Times/concept-search-api#post-conceptsearch

Below lines are all endpoints which widget requires.

```
POST https://api.ft.com/concept/search
GET https://api.ft.com/concordances?conceptId=http%3A%2F%2Fapi.ft.com%2Fthings%2F4f819dfc-ebce-4e92-94fe-dd74e4707493
GET https://api.ft.com/content?isAnnotatedBy=http%3A%2F%2Fapi.ft.com%2Fthings%2F4f819dfc-ebce-4e92-94fe-dd74e4707493
GET https://api.ft.com/content/851ac75b-8fde-3324-815c-ef987b13be95
```